org.xaraxtreme.XaraLX.json

```diff
@@ -0,0 +1,779 @@
+                        "make install -C 'contrib/src/gizmos';",
+                        "make install -C 'contrib/src/svg';"
+                    ]
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "echo 'install';",
+                        "echo 'install: begin';
+                        # install our multilib-aware wrapper
+                        #rm '/app/bin/wx-config';
+                        mv '/app/bin/wx-config' '/app/bin/wx-config-2.0';
+                        #install -p -m 755 'wx-config' '/app/bin/wx-config-2.0';
+                        touch '/app/bin/wx-config';
+                        # Alternatives setup with wxrc
+                        mv '/app/bin/wxrc' '/app/bin/wxrc-2';
+                        touch '/app/bin/wxrc';
+                        echo 'install: end';"
+                    ]
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "echo 'install docs';",
+                        "install -d '/app/share/doc/wxGTK2';
+                        install -p -m 0644 'docs/readme.txt' 'docs/readme_vms.txt' '/app/share/doc/wxGTK2/';
+                        install -p -m 0644 'docs/changes.txt' 'docs/todo.txt' '/app/share/doc/wxGTK2/';"
+                    ]
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "echo 'install licenses';",
+                        "install -d '/app/share/licenses/wxGTK2';
+                        install -p -m 0644 'docs/gpl.txt' 'docs/lgpl.txt' '/app/share/licenses/wxGTK2/';
+                        install -p -m 0644 'docs/licence.txt' 'docs/licendoc.txt' 'docs/preamble.txt' '/app/share/licenses/wxGTK2/';"
+                    ]
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "echo 'backup old autotools files';",
+                        "for filename in 'autogen' 'autogen.sh' 'configure' 'configure.ac' 'configure.in' 'install-sh' 'Makefile' 'Makefile.am' 'Makefile.in'; do
+                            [[ -f \"${filename}\" ]] || continue;
+                            mv \"${filename}\" \"${filename}.bak\";
+                        done;",
+                        "echo 'create empty Makefile';",
+                        "echo '# Empty Makefile\n\nall:\n\techo \"Using an empty Makefile doing nothing\"\n\ninstall:\n\techo \"Using an empty Makefile doing nothing\"' > 'Makefile';"
+                    ]
+                }
+            ],
+            "post-install": [
+                "echo 'post-install';",
+                "echo 'post-install: begin';
+                echo 'update-alternatives';
+                # remove symlink
+                if [[ -f '/app/bin/wx-config' ]] && [[ ! -h '/app/bin/wx-config' ]] ; then
```

Comment:
Use cleanup.

Reply:
For **update-alternatives**? I do not think so. The purpose of this code is to handle multiple versions of wxWidgets. This is useful in some applications that can use many toolkits, e.g. both wxGTK2 and wxGTK3.